src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -126,11 +126,11 @@ export class SearchResults extends Component {
       },
       'search-results-top-recommendation': undefined,
       'search-selection': 'Yellow Ribbon',
-      'search-result-chosen-page-url': window.location.href,
+      'search-result-chosen-page-url': school?.insturl || undefined,
```

Comment:
This is a business requirement change, just validating this is desired! 🙂

Reply:
Yep! I'm used the incorrect URL when I first built the object.